### DISCUSSION TITLE
can't find CRDs after going from 2.9.5 -> 2.10.4

### DISCUSSION THREAD

**Discussion by wajones3** [argoproj/argo-cd#17631] [Q&A]
running into an issue where my apps fail to sync after upgrading to v2.10.4 because they cannot find the crd
"error validating data: failed to check CRD: failed to list CRDs: the server could not find the requested resource"
I definitely have them as when I run `kubectl get crds -n argocd` on my cluster it lists the appropriate ones, namely applications.argoproj.io. We also have a clusterrole that specifically allows applications.argoproj.io the actions get, list, and watch. Any ideas?

**Comment by tken2039** [comment]
In the Argo CD that my team manages, we also observed similar incidents in some Applications. However, it was resolved by enabling `skip schema validation`, as documented in the following:

https://argo-cd.readthedocs.io/en/stable/user-guide/sync-options/#server-side-apply
